Tests/Acceptance/EpsCheckoutTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Tests\Acceptance;
+
+use Wirecard\Oxid\Model\EpsPaymentMethod;
+
+/**
+ * Acceptance tests for the Eps checkout flow.
+ */
+class EpsCheckoutTest extends CheckoutTestCase
+{
+    public function getPaymentMethod()
+    {
+        return new EpsPaymentMethod();
+    }
+
+    public function testCheckout()
+    {
+        $this->goThroughCheckout();
+        $this->goThroughExternalFlow();
+        $this->waitForRedirectConfirmation();
+
+        $this->assertPaymentSuccessful();
+    }
+
+    public function goThroughCheckout()
```

Comment:
This is identical to what's already defined in `CheckoutTestCase`.

Reply:
:+1: